meson_options.txt

```diff
@@ -15,6 +15,7 @@ option('plugin_thunderbolt', type : 'boolean', value : true, description : 'enab
 option('plugin_redfish', type : 'boolean', value : true, description : 'enable Redfish support')
 option('plugin_uefi', type : 'boolean', value : true, description : 'enable UEFI support')
 option('plugin_nvme', type : 'boolean', value : true, description : 'enable NVMe support')
+option('plugin_flashrom', type : 'boolean', value : true, description : 'enable libflashrom support')
```

Comment:
At least until the library is widely available I would prefer to keep `plugin_flashrom` disabled by default

Reply:
However turn it on for at least one of the CI builds (add an extra checkout step to a CI build if needed)